src/test/resources/features/FurtherDetailsTask.feature

```diff
@@ -136,3 +136,126 @@ Feature: Further Details Task page validation
+    And I click on Continue button
+    Then I should see error message Start date must be a real date on the Group Accounting Period Start Date Page
+    Then I should see error message Enter the end date of the group’s consolidated accounting period on the Group Accounting Period End Date Page
+    When I refresh the page
+    When Accounting Period Start Day is entered as 10
+    And Accounting Period Start Month is entered as 15
+    And Accounting Period Start Year is entered as 2024
+    And I click on Continue button
+    Then I should see error message Start date must be a real date on the Group Accounting Period Start Date Page
+    Then I should see error message Enter the end date of the group’s consolidated accounting period on the Group Accounting Period End Date Page
+    When I refresh the page
+    When Accounting Period Start Day is entered as 10
+    And Accounting Period Start Month is entered as 10
+    And Accounting Period Start Year is entered as Y2024
+    And I click on Continue button
+    Then I should see error message Start date must be a real date on the Group Accounting Period Start Date Page
+    Then I should see error message Enter the end date of the group’s consolidated accounting period on the Group Accounting Period End Date Page
+    When I refresh the page
+    When Accounting Period Start Day is entered as 10
+    And Accounting Period Start Month is entered as 15
+    And Accounting Period Start Year is entered as 20244
+    And I click on Continue button
+    Then I should see error message Start date must be a real date on the Group Accounting Period Start Date Page
+    Then I should see error message Enter the end date of the group’s consolidated accounting period on the Group Accounting Period End Date Page
+    When I refresh the page
+    When Accounting Period End Day is entered as 12
+    And I click on Continue button
+    Then I should see error message Enter the start date of the group’s consolidated accounting period on the Group Accounting Period Start Date Page
+    Then I should see error message End date must include a month and year on the Group Accounting Period End Date Page
+    When I refresh the page
+    When Accounting Period End Month is entered as 10
+    And I click on Continue button
+    Then I should see error message Enter the start date of the group’s consolidated accounting period on the Group Accounting Period Start Date Page
+    Then I should see error message End date must include a day and year on the Group Accounting Period End Date Page
+    When I refresh the page
+    When Accounting Period End Year is entered as 2024
+    And I click on Continue button
+    Then I should see error message Enter the start date of the group’s consolidated accounting period on the Group Accounting Period Start Date Page
+    Then I should see error message End date must include a day and month on the Group Accounting Period End Date Page
```

Comment:
Missing Scenarios:
End date day and month entered, but not year.
End date day and year entered, but not month.
End date month and year entered, but not year.